databricks-sdk-java/src/main/java/com/databricks/sdk/service/jobs/RunForEachTask.java

```diff
@@ -9,7 +9,7 @@
 
 @Generated
 public class RunForEachTask {
-  /** Controls the number of active iterations task runs. Default is 100 (maximal value). */
+  /** Controls the number of active iterations task runs. Default is 20, maximum allowed is 100. */
```

Comment:
This would lead to slow performance for customers if they haven't specified the default values. We should highlight this.